models/oci/oci.go

```diff
@@ -112,3 +120,105 @@ func getLayerMediaType(extension string) types.MediaType {
 	}
 	return types.MediaType(fmt.Sprintf("%s.%s", oci.CanonicalMediaTypePrefix, extension))
 }
+
+// function to pull models from any OCI-compatible repository
+func PushToOCIRegistry(dirPath, registryAdd, repositoryAdd, imageTag, username, password string) error {
+
+	// Create a new file store
+	fs, err := file.New(".")
```

Comment:
Use `~/.meshery` for temporary artifacts.

Reply:
// @RipulHandoo

Reply:
I've tried adding "./meshery" to the file path as per your suggestion, but it resulted in an error. 
`panic: failed to stat C:\Users\HP\Testing OCi registry commmand\meshery\amd-gpu: CreateFile C:\Users\HP\Testing OCi registry commmand\meshery\amd-gpu: The system cannot find the path specified.`

Reply:
What path did you enter? specify the exact path to help debug.
Since you are using mesheryctl, `~/.meshery ` folder will always be present as the mesheryctl config also lives in the same location.
Also, if that path is not present (~/.meshery), create it.

Reply:
@MUzairS15 The issue arises when accessing files from the path specified by the user with the addition of './meshery', as it disrupts the original directory structure. In this context, there's no need to include './meshery'.
![image](https://github.com/meshery/meshkit/assets/107461226/95ef9f7c-a2ed-4fc5-8ce0-1019b5db342e)

Reply:
@RipulHandoo you don’t need to append .meshery when packaging the model specified by the user.

what I am suggesting is, after you get the location of artifact user needs to push.

to package into layers and bundle it you are creating a temporary file store, so instead of creating that temporary filestore in the current directory use ..meshery for temporary file store.

Reply:
https://github.com/meshery/meshkit/pull/483#discussion_r1533538789

Did you tried this?

Reply:
Yes, I tried. Getting the same error. System can not find the file specified.